scripts/update

```diff
@@ -27,9 +27,35 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
             run --rm --no-deps app \
             yarn run build
 
-        # Bring up PostgreSQL and Django in a way that respects
-        # configured service health checks.
-        docker-compose up -d database django
+        # Make sure Postgres is available for connections before running migrations.
+        docker-compose up -d database
+
+        max=15
+        counter=1
+        while true; do
+            echo "Checking if database is up yet (try ${counter})..."
+
+            # Check if the db is ready (don't exit if this command fails)
+            set +e
+            docker-compose exec -T \
+                database gosu postgres \
+                pg_isready -U openapparelregistry > /dev/null 2>&1
```

Comment:
There are a few ways that we solve the problem of making sure that the database is healthy before running Django management commands. Since using `up` and relying on Compose health checks causes port allocation problems on CI during simultaneous builds, the two alternatives I considered were:

1. This one (use a `while` loop and `pg_isready`), adapted from [PFB](https://github.com/azavea/pfb-network-connectivity/blob/bf9a3578b519d1c906b129fcdb888e2f0bee9370/scripts/update#L22-L47)
2. `sleep` for some amount of seconds that is longer than the typical startup time for the database

Number 1) is much harder to parse given how verbose the code is, but it implements a proper health check. 2) is quicker to grok, but it relies on vague database startup heuristics that don't necessarily correspond to reality. Given these constraints, I reasoned that 1) was the better choice.